scripts/pylint_custom_plugin/pylint_guidelines_checker.py

```diff
@@ -1706,6 +1706,67 @@ def visit_functiondef(self, node):
     visit_asyncfunctiondef = visit_functiondef
 
 
+class CheckNoAliasGeneratedCode(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = "check-alias"
+    priority = -1
+    msgs = {
+        "C4745": (
+            "Aliasing and exposing generated code."
```

Comment:
small nit (sorry if this goes against what i said earlier): I think saying "Exposing aliased generated code" is slightly better